roles/ood/tasks/main.yaml

```diff
@@ -14,6 +14,43 @@
     name: "ondemand"
     state: present
 
+- name: Install dependencies-nfs-utils, ohpc rpm
+  yum: name={{ item }} state=installed update_cache=true
+  with_items:
+    - nfs-utils
+    - https://github.com/openhpc/ohpc/releases/download/v1.3.GA/ohpc-release-1.3-1.el7.x86_64.rpm
```

Comment:
What's the motivation for having the https url and nfs-utils here, ves in the following task?  In particular, shouldn't we be adding the ohpc repo as a source to the yum config and then just installing the packages like normal?

Did I miss that step?

Reply:
hunh,,,very strange, I replied here this morning but can't seem to find my comment.

ANyway, here was my reply:

So I followed the steps mentioned in [Louis's documentation](https://gitlab.rc.uab.edu/louistw/cluster-installation-note/blob/master/ood.md). He had the nfs-client install first and then rest of the installs , os I kept them in the same order in my list of packages to install. One catch with ansible is that you need to break up the list if one of the packages is dependent on another package in the same list, hence I broke it at the OHPC rpm install, as all the other RPM packages depend on it. 

I agree that nfs-utils doesn't make much sense sitting alone with the OHPC RPM install, since they are not related, os I have moved nfs-utils back to the dependencies task.

Reply:
Ok so take a look at the pre_ohpc task for doing the same thing (adding the ohpc rep) we should do the same thing.